### DISCUSSION TITLE
API Down?

### DISCUSSION THREAD

**Discussion by MaxWeichers** [revoxhere/duino-coin#1785] [Q&A]
http://51.15.127.80/api.json is not valid, i tried http://51.15.127.80:2811/ nothing is working

**Comment by sinuso** [comment]
Why IP instead of domain?
https://server.duinocoin.com/api.json

**Comment by MaxWeichers** [comment]
Because I was running it on a weird sbc and i could not get the DNS to work so I have to do the ip, I also think cloudflare is blocking the requests though
